etc/plf/setenv-customize.sh

```diff
@@ -61,6 +61,15 @@ if ${DEPLOYMENT_CHAT_ENABLED}; then
     CATALINA_OPTS="${CATALINA_OPTS} -Dchat.dbName=${EXO_CHAT_MONGODB_NAME}"
     CATALINA_OPTS="${CATALINA_OPTS} -Dchat.chatPassPhrase=${EXO_CHAT_MONGODB_NAME}"
 fi
+
+# eXo Onlyoffice addon
+if ${DEPLOYMENT_ONLYOFFICE_DOCUMENTSERVER_ENABLED}; then
+    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.host=localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}"
+    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.schema=http"
+    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.allowedhosts=localhost,${INSTANCE_KEY}.${ACCEPTANCE_HOST}"
```

Comment:
is it working if we specify a virtual host with `DEPLOYMENT_APACHE_VHOST_ALIAS ` ?